drivers/hmis/spec/models/hmis/hud/project_spec.rb

```diff
@@ -197,4 +198,15 @@ def selected_instances
       expect(selected_instances).to contain_exactly(mid_project, doe_org)
     end
   end
+
+  describe 'project custom_assessments' do
+    let!(:a1) { create(:hmis_custom_assessment, enrollment: enrollment, client: enrollment.client) }
+    let!(:a2) { create(:hmis_wip_custom_assessment, enrollment: enrollment, client: enrollment.client) }
+
+    it 'should only make 1 db query when querying for custom assessments' do
+      expect do
+        expect(project.custom_assessments.count).to eq(2), 'should return both WIP and non-WIP assessment'
```

Comment:
```suggestion
        expect(project.custom_assessments).to contain_exactly(a1, a2), 'should return both WIP and non-WIP assessment'
```
Since you're testing DB queries, its probably better to test the full query to resolve the object, rather than a `count` which will be different sql